src/ProjectAudioManager.cpp

```diff
@@ -944,28 +942,28 @@ bool ProjectAudioManager::DoRecord(AudacityProject &project,
             // ISO standard would be nice, but ":" is unsafe for file name.
             nameSuffix.Replace(wxT(":"), wxT("-"));
 
-            if (newTrack.get() == first) {
+            if (newTrack == first) {
                if (baseTrackName.empty())
                   newTrack->SetName(nameSuffix);
                else if (nameSuffix.empty())
                   newTrack->SetName(baseTrackName);
                else
                   newTrack->SetName(baseTrackName + wxT("_") + nameSuffix);
             }
-            //create a new clip with a proper name before recording is started
-            newTrack->CreateClip(t0, makeNewClipName(newTrack.get()));
 
-            if ((recordingChannels > 2) &&
-                !(ProjectSettings::Get(*p).GetTracksFitVerticallyZoomed())) {
-               ChannelView::Get(*newTrack->GetChannel(0)).SetMinimized(true);
-            }
+            //create a new clip with a proper name before recording is started
+            newTrack->CreateWideClip(t0, makeNewClipName(newTrack));
 
-            transportSequences.captureSequences.push_back(newTrack);
+            for(auto channel : newTrack->Channels())
+            {
+               ChannelView::Get(*channel).SetMinimized(minimizeChannelView);
 
-            tempList.reset();
-            trackList.RegisterPendingNewTrack(newTrack);
+               transportSequences.captureSequences.push_back(
+                  std::static_pointer_cast<WaveTrack>(channel->GetTrack().shared_from_this())
+               );
```

Comment:
If you rebase onto my fixup commit https://discord.com/channels/@me/840616766099161088/1151545897302360105, and resolve the conflict, then you will understand that the `push_back` above should be pulled out of the loop.  `captureSequences` must contain leader tracks only, and each one only once.  Then test that recording to stereo tracks still works.

Reply:
That completes review of this one commit of the draft.